components/dashboard/src/admin/License.tsx

```diff
@@ -33,11 +33,7 @@ export default function License() {
         // eslint-disable-next-line react-hooks/exhaustive-deps
     }, []);
 
-    const featureList = license?.enabledFeatures;
-    const features = license?.features;
-
-    // if user seats is 0, it means that there is no user limit in the license
-    const userLimit = license?.seats === 0 ? "Unlimited" : license?.seats;
+    const userLimit = "Unlimited";
```

Comment:
❓ do we need the License page at all? 
cc. @geropl and @atduarte

We can do that as a follow-up.